### DISCUSSION TITLE
How to stop the running test script when one of the expect is fail?

### DISCUSSION THREAD

**Discussion by BrendaAspen** [chaijs/chai#1508] [Support] [answered]
How to stop the running test script when one of the expect() is fail?

For example, I have 3 expect() items.
1 -- test result is pass
2 -- test result is **fail** 
3 -- test result is pass 

I want to stop the running test script at once when I got a fail on item 2.
But not, all expect() will be checked, and, the script can keep on running.

**Comment by keithamus** [comment, marked as answer]
This will depend on your test runner, not Chai. For example if you're using Mocha you can pass the `--bail` flag which will stop running any tests after the first failure.

**Reply by BrendaAspen** [reply]
Ok, thanks for your reply.
